constraints/match.js

```diff
@@ -112,10 +112,15 @@ function matchPS(sTree, pParent, pCat, options)
 *	minProsody: If true, the prosodic match needs to be minimal. Passed to hasMatch.
 *	nonMaxProsody: If true, the prosodic match must be non-maximal. Passed to hasMatch.
 *	nonMinProsody: If true, the prosodic match must be non-minimal. Passed to hasMatch.
+* trimTrees: Default true, trims trees of silent nodes, non-x0 terminals and redundant nodes, e.g. [[some terminals]]
```

Comment:
I think this should default to false, actually. Also we should make sure that the tableau displays whether trimming has taken place or not.

Reply:
ok, updated in commit 6ddcac2